content/docs/rfcs/35/README.md

```diff
@@ -0,0 +1,230 @@
+---
+slug: 35
+title: 35/WAKU2-NOISE
+name: Noise Protocols for Waku2 Payload Encryption
+status: raw
+editor: Giuseppe <giuseppe@status.im>
+contributors: 
+---
+
+This specification describes how Waku2 messages can be encrypted
```

Comment:
```suggestion
This specification describes how Waku Messages can be encrypted
```

Possibly with a link to tWaku message specs

Reply:
Addressed in https://github.com/vacp2p/rfc/pull/496/commits/8becd7bf7c5bf8575877b8cdc4444a4b9e5d1463